### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript-contrib/governance#35

### CODE REVIEW

**Comment by JordanMartinez** [comment]
This has a dependency on `spec` and `mmorph`.

**Comment by thomashoneyman** [comment]
While not super fun, we can migrate this to use [assert](https://github.com/purescript/purescript-assert) instead of `spec` in order to avoid having dependencies outside `contrib`:

https://github.com/purescript-contrib/purescript-unicode/blob/main/test/Test/Data/Char/Unicode.purs

I also don't see `mmorph` anywhere in the dependencies (including the downloaded packages in CI). I don't think that's actually a dependency.

**Comment by JordanMartinez** [comment]
I worked around the tests issue. How's this?

File: test/Test/Data/Char/Unicode.purs [hunk 1]
```diff
@@ -3,19 +3,45 @@ module Test.Data.Char.Unicode (dataCharUnicodeTests) where
...
+import Effect.Console (log)
+import Effect.Class (class MonadEffect, liftEffect)
 import Partial.Unsafe (unsafePartial)
 import Test.QuickCheck (quickCheck)
 import Test.QuickCheck.Arbitrary (class Arbitrary)
 import Test.QuickCheck.Gen (Gen, oneOf, chooseInt)
-import Test.Spec (Spec, describe, it)
-import Test.Spec.Assertions (shouldEqual)
+-- import Test.Spec (Spec, describe, it)
+-- import Test.Spec.Assertions (shouldEqual)
```
Comment (thomashoneyman) [inline test/Test/Data/Char/Unicode.purs]:
```suggestion
```

**Review by thomashoneyman — APPROVED** [review]

File: test/Test/Data/Char/Unicode.purs (hunk 1 above)
Comment (JordanMartinez) [reply test/Test/Data/Char/Unicode.purs]: Pushed a commit fixing this.

**Pull request merged**